.gitmodules

```diff
@@ -1,9 +1,9 @@
 [submodule "src/llvm-project"]
-	path = src/llvm-project
-	url = https://github.com/llvm/llvm-project
+        path = src/llvm-project
+        url = https://github.com/llvm/llvm-project
 [submodule "src/wasi-libc"]
-	path = src/wasi-libc
-	url = https://github.com/CraneStation/wasi-libc
+        path = src/wasi-libc
+        url = https://github.com/dicej/wasi-libc
```

Comment:
I'll switch this back once https://github.com/WebAssembly/wasi-libc/pull/429 is merged.